src/odemis/gui/util/img.py

```diff
@@ -749,17 +751,24 @@ def ar_to_export_data(streams, raw=False):
     s = streams[0]
 
     if raw:  # csv
-        pos = s.point.value
+        polpos = None
+        sempos = s.point.value  # ebeam pos selected
+        if hasattr(s, "polarization"):
+            polpos = s.polarization.value  # polarization pos selected
 
         # find positions of each acquisition
-        # tuple of 2 floats -> DataArray: position on SEM -> data
-        sempos = {}
+        # (float, float, str or None)) -> DataArray: position on SEM + polarization -> data
+        pos = {}
         for d in s.raw:
             try:
-                sempos[d.metadata[model.MD_POS]] = img.ensure2DImage(d)
+                pos[d.metadata[model.MD_POS] + (d.metadata.get(model.MD_POL_MODE, None),)] = img.ensure2DImage(d)
             except KeyError:
                 logging.info("Skipping DataArray without known position")
-        raw_ar = calculate_raw_ar(sempos[pos], s.background.value)
+
+        data = pos[sempos + (polpos,)]
+        data_bg = s._getBackground(pos[sempos + (polpos,)].metadata.get(model.MD_POL_MODE, model.MD_POL_NONE))
```

Comment:
We shouldn't access this protected function (or even have to know the format of the argument), but indeed, for now, there is no better way.
Once the StaticARStream also uses projections, this could be improved, by using the `.projectAsRaw()` function of the projection.